test/unit/test_serving.py

```diff
@@ -102,6 +103,12 @@ def test_serving_entrypoint_start_gunicorn(mock_server):
     mock_server.start.assert_called_once()
 
 
+@patch('sagemaker_xgboost_container.serving.server')
+def test_serving_entrypoint_set_default_env(mock_server):
+    serving.serving_entrypoint()
+    assert os.getenv('OMP_NUM_THREADS') == sm_env_constants.ONE_THREAD_PER_PROCESS
+
+
```

Comment:
Thanks for the test. Let's add a "negative" tests for the other entrypoints that assert `os.getenv('OMP_NUM_THREADS') != sm_env_constants.ONE_THREAD_PER_PROCESS`